java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/source/core/DbzChangeEventConsumer.java

```diff
@@ -235,6 +235,7 @@ var record = event.value();
                         // - PG: topicPrefix.schemaName.tableName
```

Comment:
Doesn't PG have the same issue?

Reply:
You can see in the comment, the pg connector doesn't prefixed with database name.
`PG: topicPrefix.schemaName.tableName`